src/Devices/Devices.js

```diff
@@ -223,8 +223,8 @@ const DevicesComponent = ({
           <Grid item xs={12} md={12} lg={12} xl={12}>
             <Typography>
               To rename a device, use the pencil icon <Edit fontSize="small" />. Add your farm code
-              in all caps anywhere in the nickname, plus any other info you want. Example: &quot;ABC
-              (Farmer Brown top of the hill)&quot;
+              in all caps anywhere in the nickname, plus any other info you want. Avoid using captial letters in the 
+              name other than your farm code. 
```

Comment:
Can we go back to having the example in addition to this new text? Also "avoid using multiple capital letters" not "avoid using captial letters" (note the spelling)